DirectProgramming/DPC++/DenseLinearAlgebra/simple-add/sample.json

```
@@ -6,5 +6,74 @@
   "languages": [{"cpp": {"properties": {"projectOptions": [{"projectType": "makefile"}]}}}],
   "targetDevice": ["CPU", "GPU", "FPGA"],
   "os": ["linux", "windows"],
-  "builder": ["ide", "make"]
+  "builder": ["ide", "make"],
+  "configurations": {
```

I think the name of the CI testing object should be more direct. The word `configurations` does not communicate to me that this contains CI tests. How about calling the object `ciTests`?
```suggestion
  "ciTests": {
```